app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -118,7 +118,8 @@ def create_submission(saved_claim)
         user_account: @current_user.user_account,
         saved_claim_id: saved_claim.id,
         auth_headers_json: auth_headers.to_json,
-        form_json: saved_claim.to_submission_data(@current_user)
+        form_json: saved_claim.to_submission_data(@current_user),
+        submit_endpoint: includes_toxic_exposure? ? :claims_api : :evss
```

Comment:
Nit: looks like the linter didn't mind but the ternary operator here is a little hard to read with the symbols, may be consider doing this in a method with if statements

Reply:
Good call - one sanity check just confirming it doesn't matter downstream if you change the type of this value?

Reply:
I believe the most common method for setting the value of an enum attribute is by using the symbolic key, but using the string representation of the symbolic key is also valid.